indico/modules/events/surveys/templates/emails/start_notification_email.txt

```diff
@@ -0,0 +1,11 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    {% trans %}Survey has started{% endtrans %}
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% block body -%}
+    Survey "{{ survey.title }}" is now open. From now on, users can submit the answers to the survey.
```

Comment:
so.. a custom message to include in the email is more a nice-to-have thing right now (just talked to @pferreir) so we don't need to spend time on it right now, but should open an issue so we won't forget it in the future